example/scripts/tests-parallel.sh

```diff
@@ -9,16 +9,13 @@ echo Running parallel integration tests
 DOT_PID="$!"
 
 TESTS=$(find spec/parallel -type f -name '*spec.js' -or -name '*Spec.js')
-
 testOutputDir=scripts/test_output
 
 rm -r -f $testOutputDir
 mkdir -p $testOutputDir
 
-./node_modules/.bin/parallel -j 4 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
-
+echo "" | ./node_modules/.bin/parallel -j 4 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
```

Comment:
Parallel requires an active console, thus this falls down in docker.    I couldn't find a Better(tm) way to make this work, short of tearing out parallel/attempting to patch it.